client/webserver/site/src/js/markets.js

```diff
@@ -858,6 +858,7 @@ export default class MarketsPage extends BasePage {
       }
     }
     this.showForm(page.verifyForm)
+    page.vPass.focus()
```

Comment:
Can we also do `page.cancelPass` in `showCancel`?

Reply:
Thanks.  I also meant to mention that with js updates, we generally bump the url query cache buster thing on entry.js at https://github.com/decred/dcrdex/blob/9d65d55c8f3b0ca613fb3a7d401346706d65d7e7/client/webserver/site/src/html/bodybuilder.tmpl#L88

and when (s)css is updated, the query on style.css at https://github.com/decred/dcrdex/blob/9d65d55c8f3b0ca613fb3a7d401346706d65d7e7/client/webserver/site/src/html/bodybuilder.tmpl#L11

Any tweak is fine but I usually grab some random chars with `openssl rand -base64 9`

Reply:
Have you considered configuring webpack to do this?   
https://powerspace.tech/how-to-cache-busting-with-webpack-5131b4af8826

Reply:
Not sure how that would work with the go html tmpl files.  I also want the token to be stable for a given git commit hash so I can see clearly what they built from.  e.g. https://github.com/decred/dcrdex/issues/843#issuecomment-730115984